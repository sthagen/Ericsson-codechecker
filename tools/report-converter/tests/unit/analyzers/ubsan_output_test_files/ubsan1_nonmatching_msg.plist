<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>diagnostics</key>
	<array>
		<dict>
			<key>category</key>
			<string>unknown</string>
			<key>check_name</key>
			<string>UndefinedBehaviorSanitizer</string>
			<key>description</key>
			<string>This is not an actual error message, just tests whether we crash on a msg we don't associate with a checker.</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>253cc81796d3eccc4c6fa0c4ade5bb53</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>5</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>4</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>5</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>4</integer>
					</dict>
					<key>message</key>
					<string>This is not an actual error message, just tests whether we crash on a msg we don't associate with a checker.</string>
				</dict>
			</array>
			<key>type</key>
			<string>ubsan</string>
		</dict>
	</array>
	<key>files</key>
	<array>
		<string>files/ubsan1.cpp</string>
	</array>
	<key>metadata</key>
	<dict>
		<key>analyzer</key>
		<dict>
			<key>name</key>
			<string>ubsan</string>
		</dict>
		<key>generated_by</key>
		<dict>
			<key>name</key>
			<string>report-converter</string>
			<key>version</key>
			<string>x.y.z</string>
		</dict>
	</dict>
</dict>
</plist>
