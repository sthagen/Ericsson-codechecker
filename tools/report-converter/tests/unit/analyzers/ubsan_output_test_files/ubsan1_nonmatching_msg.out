files/ubsan1.cpp:4:5: runtime error: This is not an actual error message, just tests whether we crash on a msg we don't associate with a checker.
